server/protocols/protocols-imap4/src/test/java/org/apache/james/imapserver/netty/IMAPServerTest.java

```diff
@@ -977,7 +977,7 @@ void loginShouldSucceed() {
                 .doesNotThrowAnyException();
         }
 
-        @Test
+        @RepeatedTest(100)
```

Comment:
debug?

Reply:
No that's intended to catch the race condition